modules/ROOT/pages/feasibility-checklists.adoc

```diff
@@ -74,57 +74,39 @@ It is also highly recommended to perform tests and benchmarks when connected dir
-If you have an existing {product-proxy} deployment, you can xref:ROOT:troubleshooting-tips.adoc#check-version[check your {product-proxy} version].
+In {product-proxy} versions earlier than 2.1.0, read-only applications require special handling to avoid connection termination due to inactivity.
 
+{company} recommends that you upgrade to version 2.1.0 or later to benefit from the heartbeat feature.
+If you have an existing {product-proxy} deployment, you can xref:ROOT:troubleshooting-tips.adoc#check-version[check your {product-proxy} version].
 For upgrade instructions, see xref:ROOT:manage-proxy-instances.adoc#_upgrade_the_proxy_version[Upgrade the proxy version].
-====
-
-//TODO: combine the below 2 sections to only use 2.1.0 or later.
-//Reconcile with troubleshooting-scenarios.adoc in case this issue is also described there.
-==== Versions older than 2.1.0
 
-If a client application only sends `SELECT` statements to a database connection then you may find that {product-proxy} terminates these read-only connections periodically, which may result in request errors if the driver is not configured to retry these requests in these conditions.
-
-This happens because {astra-db} terminates idle connections after some inactivity period (usually around 10 minutes).
-If {astra-db} is your target, and a client connection is only sending read requests to {product-proxy}, then the {astra-db} connection that is paired to that client connection will remain idle and will be eventually terminated.
-
-A potential workaround is to not connect these read-only client applications to {product-proxy}, but you need to ensure that these client applications switch reads to the target at any point after all the data has been migrated and all validation and reconciliation has completed.
-
-Another work around is to implement a mechanism in your client application that creates a new `Session` periodically to avoid the {astra-db} inactivity timeout.
-You can also implement some kind of meaningless write request that the application sends periodically to make sure the {astra-db} connection doesn't idle.
-
-==== Version 2.1.0 and newer
-
-This issue is solved in version 2.1.0 of {product-proxy}, which introduces periodic heartbeats to keep alive idle cluster connections.
-We strongly recommend using version 2.1.0 (or newer) to benefit from this improvement, especially if you have a read-only workload.
+If you cannot upgrade to version 2.1.0 or later, see the alternatives described in xref:ROOT:troubleshooting-tips.adoc#client-application-closed-connection-errors-every-10-minutes-when-migrating-to-astra-db[Client application closed connection errors every 10 minutes when migrating to {astra-db}].
 
 [[non-idempotent-operations]]
 == Lightweight Transactions and other non-idempotent operations
 
 Examples of non-idempotent operations in CQL are:
 
-* Lightweight Transactions (LWTs)
+* Lightweight Transactions (LWTs) (see <<_lightweight_transactions_and_the_applied_flag>>)
```

Comment:
here and elsewhere, sometimes "Lightweight Transaction" is lowercase "transactions". Not sure which one is correct.